exec/java-exec/src/main/java/org/apache/drill/exec/server/rest/profile/ProfileResources.java

```diff
@@ -77,12 +77,14 @@
    */
   public static String getPrettyDuration(long startTimeMillis, long endTimeMillis) {
     long durationInMillis = (startTimeMillis > endTimeMillis ? System.currentTimeMillis() : endTimeMillis) - startTimeMillis;
-    long hours = TimeUnit.MILLISECONDS.toHours(durationInMillis);
+    long days = TimeUnit.MILLISECONDS.toDays(durationInMillis);
```

Comment:
Initializing an object for rendering compact or verbose duration formats might be an overkill. 
I'll change the signature for the existing method _getPrettyDuration_ in the _utility_ class - `ProfileResources.java` to carry an additional parameter to indicating verbose or compact representation and apply that in DRILL-5230. I can rebase this commit on top of that.